README.md

```diff
@@ -0,0 +1,20 @@
+
+
+# How to run test cases
+
+1. compile the code by running:
+
+   make
+
+
+2. run the test case in the folder tests, for example running the test case t1.so in the folder "tests"
+
+   krun tests/t1.sol
+
+
+3. If the test case is passed then the cell <mode/> in the configuation is success else
+   the cell <k/> is stuck at some unfinished term.
+
```

Comment:
@dfilaretti, I still have the following warning message
```
[Warning] Critical: Found 0 integer variables in exit code pattern. Returning
111
```

Do you know how to get rid of it?

Reply:
Start the `<exit-code>` cell with a `0` instead of a `1` in it if you want to disable it completely.

But really, what should happen is when a test passes it should set `<exit-code>` to zero, so that you know it passed. This is how KEVM does it. So basically, there is a rule which goes something like:

```
<k> success => . ... </k>
<exit-code> _ => 0 </exit-code>
```

Reply:
@shangweilin it looks like the exit-code cell hasn't been started with `0`, and is not being used yet.

Eventually, it's much quicker to run tests using the exit-code as a success metric than by comparing output (because then you need to open a file on disk and write to it), which will be needed once we are executing lots of tests. For now, I think just set it to `0` in the configuration so that it's not printing the annoying error message.